src/rgw/rgw_sal_motr.h

```diff
@@ -22,6 +22,8 @@ extern "C" {
 #pragma clang diagnostic ignored "-Wextern-c-compat"
 #include "motr/config.h"
 #include "motr/client.h"
+#include "lib/thread.h"
```

Comment:
Shouldn't it be `"motr/lib/thread.h"` ?

Reply:
No, just `lib/thread`. It's needed for adoptation of non-motr thread for ADDB logging.